integration-tests/maven/build.gradle.kts

```diff
@@ -16,10 +14,13 @@ dependencies {
 tasks.integrationTest {
     dependsOnMavenLocalPublication()
 
-    val setupMavenTask = project(":runners:maven-plugin").tasks.withType<SetupMaven>().single()
-    dependsOn(setupMavenTask)
```

Comment:
Instead of using a cross-project dependency to re-use the Maven exec, just download it again.

Gradle will cache the downloaded dependency, so it's fine to set it up twice in different subprojects.